tests/test_catalog.py

```diff
@@ -1,19 +1,18 @@
 import json
 import os
+import pathlib
 import tempfile
-from pathlib import Path
 
 import geopandas as gpd  # type: ignore
 import pandas as pd
 import pytest
 
-from up42.catalog import Catalog
-from up42.order import Order
+from up42 import catalog, order
 
-from .fixtures.fixtures_globals import API_HOST, DATA_PRODUCT_ID, ORDER_ID, WORKSPACE_ID
+from .fixtures import fixtures_globals
```

Comment:
NIT: as globals?

Reply:
pylint complains with
`W0622: Redefining built-in 'globals' (redefined-builtin)`

I will import it as `constants`